src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V2/DownloadedPackagesSQLLiteStoreV2.swift

```diff
@@ -58,7 +58,11 @@ extension DownloadedPackagesSQLLiteStoreV2: DownloadedPackagesStoreV2 {
 
 	func open() { // might throw errors in future versions!
 		queue.sync {
-			self.database.open()
+			
+			guard self.database.open() else {
+				Log.error("Open database error", log: .localData, error: self.database.lastError())
```

Comment:
Opening*

Reply:
@naveeddotio What exactly do you want changed here?

Reply:
Error message